src/components/CalendarMonthGrid.jsx

```diff
@@ -204,6 +216,32 @@ class CalendarMonthGrid extends React.Component {
     onMonthTransitionEnd();
   }
 
+  onMonthSelect(currentMonth, newMonthVal) {
```

Comment:
`currentMonth` is a momentObject, whereas `newMonthVal` is a zero-indexed integer?